### PR TITLE
Decreased runway light thump sound, improved ground roll, and small fixes

### PR DESCRIPTION
**Pull request by hotshotp**
## Summary of Changes

I made the runway light thunks quieter since they were overly loud before, especially on landing. they shouldn't sound awfully loud anymore.

I added another ground roll sound that sounds like the nosewheel vibration that gets higher pitched. only heard in cockpit and front cabin - it's pretty quiet, but nevertheless a pretty important part of it.

I made the seatbelt sign and cabin call sounds have a more gritty sound to them, like they are coming out of a speaker rather than just having a sine wave play. It is a very small change also, barely noticeable in the cockpit with noise and i tried not to overdo it.

Changed the battery relay sound to be more realistic and "clunky" rather than "slappy" - remember, BATTERY relay, not external power

Finally, added back the FWC triple click 45 seconds after startup

## References

https://www.youtube.com/watch?v=eWRtBE78LUM wheel vibration - starting at around 1:14 (not a neo but same wheel)

## Additional context

Discord username (if different from GitHub):

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created and uploaded.
The build script will have already been run with the latest changes, so no need to rerun it once you download the zip.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the right side, click on the **Artifacts** drop down and click the **A32NX** link

### CODE REVIEW

File: A32NX/SimObjects/AirPlanes/Asobo_A320_NEO/sound/sound.xml [hunk 1]
```diff
@@ -150,14 +150,21 @@
...
                 <Range UpperBound="0" />
             </Requires>
         </Sound>
 
+        <Sound WwiseEvent="FMGCtest" WwiseData="true" NodeName="PEDALS_LEFT" Continuous="false" SimVar="ELECTRICAL MAIN BUS VOLTAGE" Units="VOLTS" Index="1">
+            <Range LowerBound="28" />
+            <Requires LocalVar="A32NX_AVIONICS_STARTUP_SOUNDS_INHIBIT" Units="Bool" Index="0">
+                <Range UpperBound="0" />
+            </Requires >
+            </Sound>
```
Comment (Benjozork) [inline A32NX/SimObjects/AirPlanes/Asobo_A320_NEO/sound/sound.xml:166]: Indentation

File: A32NX/SimObjects/AirPlanes/Asobo_A320_NEO/sound/sound.xml [hunk 2]
```diff
@@ -589,6 +596,15 @@
...
         </Sound>
+        
+        <Sound WwiseData="true" WwiseEvent="groundrollnose" NodeName="PEDALS_LEFT" ViewPoint="Inside" FadeOutType="2" FadeOutTime="1" SimVar="GEAR ANIMATION POSITION" Units="PERCENT" Index="0">
+            <Range LowerBound="55" />
+            <Requires SimVar="GEAR TOTAL PCT EXTENDED" Units="PERCENT" Index="0">
+                <Range LowerBound="1" />
+            </Requires>
+            <WwiseRTPC SimVar="GROUND VELOCITY" Units="KNOTS" Index="0" RTPCName="SIMVAR_GROUND_VELOCITY" />
+        </Sound>
+
```
Comment (Benjozork) [inline A32NX/SimObjects/AirPlanes/Asobo_A320_NEO/sound/sound.xml:607]:
```suggestion
```

**Review by Benjozork — COMMENTED** [review]
LGTM

**Comment by MapleNAV-TCASFAULT** [comment]
Category: QA Tester
Name: ChristopherXie#3904
Date of testing: Current date 12/19/2020
Version of the sim: Version the tester uses
PR Tested: #2553 
Object of test: sounds

Testing technique: spawned on the runway and at the gate CnD
Steps you've made to test the Pull Request:
1. Spawned on the runway and took off, the overly loud light thump was fixed, but the ground roll was very minor with almost no change IMO (not a big issue though)
2. Tested the seatbelt and no smoking sounds
3. Spawned cold and dark to listen to the new battery relays, nice change
4.
5.

Flight notes (If required):

Issues (If Any): 
Overall Rating: Good, but could be slightly improved
Conclusions of the testing: Very small PR, I like the changes, everything works like it should, good job Boris.

**Comment by sidnov** [comment]
Category: QA Tester
Name: sidnov#8337
Date of testing: 20/12/2020
Version of the sim: 1.11.7.0
PR Tested: #2553
Object of test: Checking the updated sounds in different parts of the A/C
Testing technique: Cold and Dark/ Runway takeoff and circuit
Steps you've made to test the Pull Request:
1. Spawned C/D at VABB
2. Checked battery relay sounds, seatbelt signs. Worked as expected.
3. Spawned again at VABB RWY27 for departure
4. Checked the rolling sounds, looks good.
5. Took off, flew a circuit and back to land at 27.
6. Checked rollout sound, with no reverse thrust and low autobrakes. Sounds more balanced now.


Issues (If Any): None
Overall Rating: Good
Conclusions of the testing: Sounds are a really important part of the whole immersive experience, this is a pretty good improvement. Good to go!

**Comment by tjburni** [comment]
Category: QA Tester
Name:[Z+1]tjburni#0409
Date of testing: Current date 20.12.2020
Version of the sim: 1.11.7
PR Tested: #2553
Object of test: Decreased runway light thump sound, improved ground roll, and small fixes

Testing technique: Description of how you tested the Pull Request
Steps you've made to test the Pull Request:
1. Started C&D
2. while taxiing and while takeoff heared to the sounds and compared to sounds before
3. took off
4.landed
5.

Flight notes (If required):

Issues (If Any): none
Overall Rating: Good
Conclusions of the testing: Overall it feels more realistic now. Good Job. Thanks

**Review by Benjozork — APPROVED** [review]
LGTM

**Pull request merged**